### PR TITLE
randomprime v1.25.0

### PR DESCRIPTION
**Pull request by toasterparty**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -78,6 +78,19 @@ Fixed: Seeds with more than 3 starting Energy Parts can be exported and played.
 
 ### Metroid Prime
 
+- Fixed: Crash in Central Dynamo/Quarantine Access A due to the memory from extra blast shields
+- Fixed: Crash in Deck Beta Security Hall due to the memory from extra blast shields and auto-loads
+- Fixed: Non-NTSC 0-00 Ruined Courtyard thermal conduit patch exploit where Super Missiles could be skipped
+- Fixed: Upper Research Core door forcefield having the wrong color/vulnerability when a custom blast shield is placed
+- Fixed: Upper Research Core door opening if a custom blast shield is destroyed while door is unpowered
```
Comment (Miepee) [inline CHANGELOG.md:85]:
```suggestion
- Fixed: Upper Research Core door opening if a custom blast shield is destroyed while the door is unpowered
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -78,6 +78,19 @@ Fixed: Seeds with more than 3 starting Energy Parts can be exported and played.
...
+- Fixed: Export error when using (deprecated) Major cutscene skips
+- Fixed: Shorelines lighthouse cutscene skip not working if the player had visited the save station
+- Fixed: Missing black bars in Elite Research bottom platform activation cutscene
```
Comment (Miepee) [inline CHANGELOG.md:88]:
```suggestion
- Fixed: Missing black bars in the Elite Research bottom platform activation cutscene
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -78,6 +78,19 @@ Fixed: Seeds with more than 3 starting Energy Parts can be exported and played.
...
+- Fixed: Black bars sometimes disappearing prematurely in Chozo Ice Temple cutscene
```
Comment (Miepee) [inline CHANGELOG.md:89]:
```suggestion
- Fixed: Black bars sometimes disappearing prematurely in the Chozo Ice Temple cutscene
```

**Review by Miepee — APPROVED** [review]
On a side note, im surprised the shorelines fix only made it in now. feels like it was so much longer since i saw the PR.

**Comment by toasterparty** [comment]
There have been multiple issues in shorelines

**Comment by Miepee** [comment]
meant the cutscene skip one.

**Pull request merged**